src/System.Net.Primitives/src/project.json

```diff
@@ -0,0 +1,27 @@
+{
+  "dependencies": {
+    "System.Collections": "4.0.0-beta-*",
+    "System.Diagnostics.Contracts": "4.0.0-beta-*",
+    "System.Diagnostics.Tracing": "4.0.0-beta-*",
```

Comment:
@pgavlin When I try to compile this I'm getting

Restoring all packages...
S:\c2\build.proj(24,5): error : Unable to locate System.Diagnostics.Tracing 4.0.0-beta-23127
S:\c2\build.proj(24,5): error : Unable to locate System.Diagnostics.Tracing 4.0.0-beta-23127
S:\c2\build.proj(24,5): error : Unable to locate System.Diagnostics.Tracing 4.0.0-beta-23127
  Errors in S:\c2\src\System.Net.Primitives\src\project.json
      Unable to locate System.Diagnostics.Tracing 4.0.0-beta-23127
  Errors in S:\c2\src\System.Net.Primitives\tests\FunctionalTests\project.json
      Unable to locate System.Diagnostics.Tracing 4.0.0-beta-23127
  Errors in S:\c2\src\System.Net.Primitives\tests\UnitTests\project.json
      Unable to locate System.Diagnostics.Tracing 4.0.0-beta-23127
S:\c2\build.proj(24,5): error MSB3073: The command ""S:\c2\packages//dnx-coreclr-win-x86.1.0.0-beta5-12101/\bin\dnu.cmd" restore
--packages "S:\c2\packages" --source https://www.myget.org/F/dotnet-core/ --source https://www.myget.org/F/dotnet-coreclr/ --sour
ce https://www.myget.org/F/dotnet-corefx/ --source https://www.myget.org/F/dotnet-corefxtestdata/ --source https://www.myget.org/
F/dotnet-buildtools/ --source https://www.nuget.org/api/v2/ "S:\c2\src" "S:\c2\packages/Microsoft.DotNet.BuildTools.1.0.25-prerel
ease-00064/lib/"" exited with code 1.